internal/provider/bootstrap.go

```diff
@@ -59,35 +57,18 @@ func Bootstrap(e *pluggable.Event) pluggable.EventResponse {
 
 	logLevel := "debug"
 
-	if p2pBlockDefined && providerConfig.P2P.LogLevel != "" {
-		logLevel = providerConfig.P2P.LogLevel
-	}
-
-	lvl, err := logging.LevelFromString(logLevel)
-	if err != nil {
-		return ErrorEvent("Failed setup logger: %s", err.Error())
+	if p2pBlockDefined && prvConfig.P2P.LogLevel != "" {
+		logLevel = prvConfig.P2P.LogLevel
 	}
 
-	// TODO: Fixup Logging to file
-	loggerCfg := zap.NewProductionConfig()
-	loggerCfg.OutputPaths = []string{
-		cfg.Logfile,
-	}
-	logger, err := loggerCfg.Build()
-	if err != nil {
-		return ErrorEvent("Failed setup logger: %s", err.Error())
-	}
-
-	logging.SetAllLoggers(lvl)
-
-	log := &logging.ZapEventLogger{SugaredLogger: *logger.Sugar()}
+	logger := types.NewKairosLogger("agent", logLevel, false)
```

Comment:
this is the main change, dropping the zap logger for our kairos generic logger

Reply:
Does our logger log to a file too? It would be nice if that was configured centrally somewhere. Obviously it doesn't make sense to handle this everywhere we create a logger (env var maybe?). Not for this PR though.

Reply:
It does. It tries to create files both to /run/Kairos and /var/log/Kairos so it would write logs even if for some reason /var/log is immutable, we still got the /run/Kairos logs 


Ideally a couple of methods to pass along your own log locations should be added so consumers can set their own locations but it didn't seem necessary at the time so I postponed adding it for now